src/mantine-hooks/src/utils/is-function/is-function.ts

```diff
@@ -0,0 +1,3 @@
+export function isFunction(x: unknown): x is Function {
```

Comment:
Please remove this

Reply:
In such a case, I can use `action instanceof Function` inside the hook (but this is a workaround), since there is a known issue with checking such type with `typeof` https://github.com/microsoft/TypeScript/issues/37663

If it is okay I will remove this

Reply:
typeof should work just fine

Reply:
Unfortunately no
<img width="722" alt="ts" src="https://user-images.githubusercontent.com/51407533/200797674-9a98e655-e451-4b5f-8152-e036b69aa751.png">

Reply:
okay, then use instance of